src/clover/index.ts

```diff
@@ -61,28 +64,34 @@ export const fromString = (str: string): Stats => {
       branchs: new Coverage(m.conditionals, m.coveredconditionals),
     },
     allFiles
```

Comment:
just to reduce the changes on the file, you could do this:
```suggestion
    allFiles
      .map(f => {
        f._attributes.name = file._attributes.path || file._attributes.name
        return f
      })
```

doing that the other changes of this map-reduce can be reverted.

Reply:
@lucassabreu Improved in https://github.com/lucassabreu/comment-coverage-clover/pull/27/commits/4d89722e7978febead5047ddbf4d1c6c9edce8b4